libs/platform/src/lib/table/table.component.ts

```diff
@@ -1404,7 +1404,15 @@ export class TableComponent<T = any> extends Table<T> implements AfterViewInit,
             return;
         }
 
-        event?.preventDefault();
+        if (event) {
```

Comment:
@mikerodonnell89 what if we check whether the row itself is currently focused, instead of checking the element type?
I mean, when user presses enter on focused row, that's when the navigation, or activation should happen. If user focuses inside the row, then he can press any button he likes. What do you think?

Reply:
The only time this function is ever called and an event is actually passed is when a space press is made. But I can be more explicit in this function here and check specifically for the space press